requirements/base.txt

```diff
@@ -197,9 +200,9 @@ markupsafe==2.1.2
     # via jinja2
 maykin-django-two-factor-auth[phonenumbers]==2.0.4
     # via -r requirements/base.in
-mozilla-django-oidc==2.0.0
+mozilla-django-oidc==4.0.0
     # via mozilla-django-oidc-db
-mozilla-django-oidc-db==0.12.0
+mozilla-django-oidc-db==0.14.1
```

Comment:
@annashamray because this version of mozilla-django-oidc-db uses `django-jsonform`, it has to be added to the installed apps: https://github.com/maykinmedia/objects-api/pull/360